src/models/index.js

```diff
@@ -4,7 +4,7 @@ const Sequelize = require('sequelize');
 
 const basename = path.basename(__filename);
 const env = process.env.NODE_ENV || 'development';
-const config = require('../../config/databaseConfig')[env];
+const config = require('../config/databaseConfig')[env];
```

Comment:
@chikeozulumba why do we need to the structure of config ?

Reply:
This structure is because i moved the models folder to the root of the src